tests/e2e/util/common/e2e_utils.go

```diff
@@ -261,3 +263,60 @@ func InstallOperatorViaHelm(extraArgs ...string) error {
 func UninstallOperator() error {
 	return helm.Uninstall("sail-operator", "--namespace", OperatorNamespace)
 }
+
+// GetYAMLPodURL returns the URL of the yaml file for the testing app.
+// args:
+// version: the version of the Istio to get the yaml file from.
+// appName: the name of the testing app. Example: helloworld, sleep, tcp-echo.
+func GetYAMLPodURL(version supportedversion.VersionInfo, appName string) string {
```

Comment:
I know it was called that before, but `GetYAMLPodURL` is a strange name for this func. Isn't  `GetSampleYAML` more adequate? Not sure what the pod has to do with this?